#!/bin/sh /usr/share/dpatch/dpatch-run
## 02_disable_epn.dpatch by Jan Wagner <waja@cyconet.org>
##
## DP: leave marks to disable epn

@DPATCH@

diff -Nur nagios_plugins.orig/check_snmp_boostedge.pl nagios_plugins/check_snmp_boostedge.pl
--- nagios_plugins.orig/check_snmp_boostedge.pl	2007-01-16 23:57:14.000000000 +0100
+++ nagios_plugins/check_snmp_boostedge.pl	2008-02-10 20:41:52.345434535 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_boostedge.pl #################
 # Version : 1.0
 # Date : Jan 16 2007
diff -Nur nagios_plugins.orig/check_snmp_cpfw.pl nagios_plugins/check_snmp_cpfw.pl
--- nagios_plugins.orig/check_snmp_cpfw.pl	2007-04-20 00:03:30.000000000 +0200
+++ nagios_plugins/check_snmp_cpfw.pl	2008-02-10 20:42:02.055266874 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_cpfw ##############
 # Version : 1.2.1
 # Date : April 19 2007
diff -Nur nagios_plugins.orig/check_snmp_css.pl nagios_plugins/check_snmp_css.pl
--- nagios_plugins.orig/check_snmp_css.pl	2007-01-11 17:06:08.000000000 +0100
+++ nagios_plugins/check_snmp_css.pl	2008-02-10 20:42:10.943113408 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_css.pl #################
 # Version : 1.0.1
 # Date : 27 Sept 2006
diff -Nur nagios_plugins.orig/check_snmp_css_main.pl nagios_plugins/check_snmp_css_main.pl
--- nagios_plugins.orig/check_snmp_css_main.pl	2006-10-05 23:40:55.000000000 +0200
+++ nagios_plugins/check_snmp_css_main.pl	2008-02-10 20:42:19.513965416 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_css_main.pl #################
 # Version : 1.0	
 # Date : 27 Sept 2006
diff -Nur nagios_plugins.orig/check_snmp_env.pl nagios_plugins/check_snmp_env.pl
--- nagios_plugins.orig/check_snmp_env.pl	2007-04-20 11:45:22.000000000 +0200
+++ nagios_plugins/check_snmp_env.pl	2008-02-10 20:42:25.912854927 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_env #################
 # Version : 1.2
 # Date : April 19 2007
diff -Nur nagios_plugins.orig/check_snmp_int.pl nagios_plugins/check_snmp_int.pl
--- nagios_plugins.orig/check_snmp_int.pl	2007-04-23 11:40:39.000000000 +0200
+++ nagios_plugins/check_snmp_int.pl	2008-02-10 20:42:33.816718453 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_int ##############
 # Version : 1.4.6
 # Date : April 23 2007
diff -Nur nagios_plugins.orig/check_snmp_linkproof_nhr.pl nagios_plugins/check_snmp_linkproof_nhr.pl
--- nagios_plugins.orig/check_snmp_linkproof_nhr.pl	2006-08-26 14:37:12.000000000 +0200
+++ nagios_plugins/check_snmp_linkproof_nhr.pl	2008-02-10 20:42:41.965577749 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_linkproof_nhr #################
 # Version : 1.0
 # Date : Aug 24 2006
diff -Nur nagios_plugins.orig/check_snmp_load.pl nagios_plugins/check_snmp_load.pl
--- nagios_plugins.orig/check_snmp_load.pl	2007-04-16 19:41:20.000000000 +0200
+++ nagios_plugins/check_snmp_load.pl	2008-02-10 20:42:49.700444194 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_load #################
 # Version : 1.3.2
 # Date : Jan 16 2007
diff -Nur nagios_plugins.orig/check_snmp_mem.pl nagios_plugins/check_snmp_mem.pl
--- nagios_plugins.orig/check_snmp_mem.pl	2006-07-09 22:52:37.000000000 +0200
+++ nagios_plugins/check_snmp_mem.pl	2008-02-10 20:42:58.441293269 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_mem ##############
 # Version : 1.1
 # Date : Jul 09 2006
diff -Nur nagios_plugins.orig/check_snmp_nsbox.pl nagios_plugins/check_snmp_nsbox.pl
--- nagios_plugins.orig/check_snmp_nsbox.pl	2007-01-16 23:55:21.000000000 +0100
+++ nagios_plugins/check_snmp_nsbox.pl	2008-02-10 20:43:06.286157817 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_nsbox #################
 # Version : 1.0
 # Date : Jan 16 2007
diff -Nur nagios_plugins.orig/check_snmp_process.pl nagios_plugins/check_snmp_process.pl
--- nagios_plugins.orig/check_snmp_process.pl	2007-03-12 21:36:02.000000000 +0100
+++ nagios_plugins/check_snmp_process.pl	2008-02-10 20:43:15.957990819 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_process ##############
 # Version : 1.4
 # Date : March 12 2007
diff -Nur nagios_plugins.orig/check_snmp_storage.pl nagios_plugins/check_snmp_storage.pl
--- nagios_plugins.orig/check_snmp_storage.pl	2007-03-12 22:01:36.000000000 +0100
+++ nagios_plugins/check_snmp_storage.pl	2008-02-10 20:43:23.024868799 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_storage ##############
 # Version : 1.3.2
 # Date :  March 12 2007
diff -Nur nagios_plugins.orig/check_snmp_vrrp.pl nagios_plugins/check_snmp_vrrp.pl
--- nagios_plugins.orig/check_snmp_vrrp.pl	2006-08-23 23:39:48.000000000 +0200
+++ nagios_plugins/check_snmp_vrrp.pl	2008-02-10 20:43:30.870733331 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_vrrp ##############
 # Version : 1.3
 # Date : Aug 23 2006
diff -Nur nagios_plugins.orig/check_snmp_win.pl nagios_plugins/check_snmp_win.pl
--- nagios_plugins.orig/check_snmp_win.pl	2006-11-29 20:59:55.000000000 +0100
+++ nagios_plugins/check_snmp_win.pl	2008-02-10 20:43:40.407568666 +0100
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_win ##############
 # Version : 0.6
 # Date : Nov 29 2006 
